src/services/merchantWebhookQueue.service.ts

```
@@ -78,63 +78,144 @@ export class MerchantWebhookQueueService {
+    if (nextRetry !== null) {
+      updateData.nextRetry = nextRetry;
+    }
+
+    if (completedAt) {
+      updateData.completedAt = completedAt;
+    }
+
+    try {
+      await this.merchantWebhookEventRepository.update(
+        { jobId: jobId },
+        updateData as any
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)